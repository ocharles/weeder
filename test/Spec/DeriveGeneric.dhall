{ roots = [ "Spec.DeriveGeneric.DeriveGeneric.t" ], type-class-roots = False }
