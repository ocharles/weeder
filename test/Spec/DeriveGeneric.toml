roots = [ "Spec.DeriveGeneric.DeriveGeneric.t" ]

type-class-roots = false
