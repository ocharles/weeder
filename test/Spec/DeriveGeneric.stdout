main: test/Spec/DeriveGeneric/DeriveGeneric.hs:12:7: (Instance) :: FromJSON T
